packages/inquirer-gui/__tests__/navigator.spec.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-undef */
```

Comment:
so as we discussed - no need to add the test here. We'll have 2 coverage reports.. 1 for the inquirer gui and the other one for your control... so you can remove it from here